mesheryctl/cmd/perf.go

```diff
@@ -32,7 +32,7 @@ var (
 	testMesh = ""
 	// testFile         = ""
 	qps              = ""
-	concurrentRequests = ""
+	parallelRequests = ""
```

Comment:
@chandresh-pancholi in order to provide consistency of user experience, we'll want to mimic the language found in the UI here in the CLI. I believe we'll want this parameter to be named "concurrentRequests". @peterj @ashiscs did we converge on "concurrentRequests"?

@chandresh-pancholi great job in following the design outlined in our [cli doc](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.45f30knxz69z).

Reply:
Yes sir we continued with concurrentRequests!!